lib/httpInterface.js

```diff
@@ -675,8 +675,7 @@ async function injectContextInRequest(request) {
     userId = userIdHeader
   }
 
-  request.context = {
-    ...request.context,
+  request.crudContext = {
```

Comment:
I assume this change is to separate the Fastify request context from the one dedicated to CRUD Service. Is this correct?

Reply:
Yes, correct. This is required since from fastify v4 it raise a deprecation warning on read context (which will be removed in fastify v5) and throws an error setting a context, since it has not a setter method.